### ISSUE TITLE
Warning when using with dry-core 0.2

### ISSUE THREAD

**Issue by fnordfish** [dry-rb/dry-validation#276]
After a "bundle update" my dry-core got updated from 0.1.0 to 0.2.1 and this warning started to appear:

```
/home/vagrant/.gem/ruby/2.3.1/gems/dry-validation-0.10.3/lib/dry/validation/message_compiler.rb:13: warning: already initialized constant Dry::Validation::MessageCompiler::EMPTY_OPTS
/home/vagrant/.gem/ruby/2.3.1/gems/dry-core-0.2.1/lib/dry/core/constants.rb:52: warning: previous definition of EMPTY_OPTS was here
```

Current work-around:

Manually downgraded to dry-core 0.1.0 using `gem "dry-core", "~> 0.1", "< 0.2"`

**Comment by flash-gordon** [comment]
Yeah, I've pushed a new version of dry-core today, it's backward compatible overall but causes these warnings in current dry-v release. @solnic could you release minor updates of dry-t and dry-v (I've updated master branches just after the release)?

**Comment by flash-gordon** [comment]
@solnic just a kindly reminder :)

**Comment by solnic** [comment]
OK I'll push releases on Friday

**Comment by jcoyne** [comment]
<img src="https://www.friday.ie/wp-content/themes/friday/img/logo_large.png">

**Comment by solnic** [comment]
https://rubygems.org/gems/dry-types/versions/0.9.3
https://rubygems.org/gems/dry-validation/versions/0.10.4

closing

**Comment by mjgiarlo** [comment]
:tada:

Thanks, @solnic!

**Comment by solnic** [comment]
such release no warnings wow ;)

**Comment by mjgiarlo** [comment]
I miss them already.
